docs/standard/design-guidelines/choosing-between-anonymous-and-tuple.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Choosing between anonymous and tuple types
+description: Learn how to decide whether to design a type as a class, or to design a type as a struct.
```

Comment:
@IEvangelist sorry for the late feedback, but this description doesn't match the article

Reply:
Ah, yes! Good catch... updating now.